docs/get-started.md

```diff
@@ -1,189 +1,77 @@
-         Gemini API using Google AI Studio. Then, set the `GOOGLE_GENAI_API_KEY`
-         environment variable to your key:
-
-         ```posix-terminal
-         export GOOGLE_GENAI_API_KEY=<your API key>
-         ```
+```
+export GOOGLE_GENAI_API_KEY=<your API key>
+```
 
-       - {Gemini (Vertex AI)}
-
-         If the Google AI Gemini API is not available in your region, consider
-         using the Vertex AI API which also offers Gemini and other models. You
-         will need to have a billing-enabled Google Cloud project, enable AI
-         Platform API, and set some additional environment variables:
-
-         ```posix-terminal
-         gcloud services enable aiplatform.googleapis.com
-
-         export GCLOUD_PROJECT=<your project ID>
+Note: While this tutorial uses the Gemini API from AI Studio, Genkit supports a wide variety of model providers including [Gemini from Vertex AI](https://firebase.google.com/docs/genkit/plugins/vertex-ai#generative_ai_models), Anthropic’s Claude 3 models and Llama 3.1 through the [Vertex AI Model Garden](https://firebase.google.com/docs/genkit/plugins/vertex-ai#anthropic_claude_3_on_vertex_ai_model_garden), open source models through [Ollama](https://firebase.google.com/docs/genkit/plugins/ollama), and several other [community-supported providers](https://firebase.google.com/docs/genkit/models#models-supported) like OpenAI and Cohere.
 
-         export GCLOUD_LOCATION=us-central1
-         ```
+## Import the library
 
-         See https://cloud.google.com/vertex-ai/generative-ai/pricing for Vertex AI pricing.
-
-    1. Choose default answers to the rest of the questions, which will
-       initialize your project folder with some sample code.
+Import the Genkit core libraries and the plugin for the Google AI Gemini APIs.
 
-    The `genkit init` command creates a sample source file, `index.ts`, which
-    defines a single flow, `menuSuggestionFlow`, that prompts an LLM to suggest
-    an item for a restaurant with a given theme.
+```javascript
+import { generate } from '@genkit-ai/ai';
+import { configureGenkit } from '@genkit-ai/core';
+import { googleAI, gemini15Flash } from '@genkit-ai/googleai';
+```
 
-    This file looks something like the following (the plugin configuration steps
-    might look different if you selected Vertex AI):
+## Make your first request
 
-    ```ts
-    import * as z from 'zod';
+Use the `generate` method to generate a text response.
 
-    // Import the Genkit core libraries and plugins.
-    import { generate } from '@genkit-ai/ai';
-    import { configureGenkit } from '@genkit-ai/core';
-    import { defineFlow, startFlowsServer } from '@genkit-ai/flow';
-    import { googleAI } from '@genkit-ai/googleai';
+```javascript
+// Make sure to include these imports:
+// import { generate } from '@genkit-ai/ai';
+// import { configureGenkit } from '@genkit-ai/core';
+// import { googleAI, gemini15Flash } from '@genkit-ai/googleai';
```

Comment:
nit: I'd probably just leave these uncommented, it's easier for the dev to simply not copy them if they already have them, vs. copying then having to uncomment each one.